src/app/components/module-index-page/module-index-page.component.ts

```diff
@@ -66,12 +45,11 @@ export class ModuleIndexPageComponent implements OnInit {
                if (response != null) {
                   this.parseContentResponse(response, module);
                } else {
-                  this.toastr.error('Response was null');
+                  alert('Response was null');
```

Comment:
Were changes is dev pulled? These alerts have been changed and are trying to be reverted

Reply:
Toastr should be re-implemented now.